app/models/spree/address_decorator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Spree
+  Address.class_eval do
```

Comment:
The original Spree::Address has no such method like `update_attributes_from_params`, so maybe we could do without `class_eval`, just opening the class?